Eastern-Research-Group/mywaterway
# Feature/468 discharger text changes

## Related Issues:
* [HMW-467](https://jira.epa.gov/browse/HMW-467)
* [HMW-468](https://jira.epa.gov/browse/HMW-468)
* [HMW-470](https://jira.epa.gov/browse/HMW-470)
* [HMW-471](https://jira.epa.gov/browse/HMW-471)

## Main Changes:
* 467 - Linked waterbody conditions to glossary
* 468 
  * Updated description saved to AGO such that "Note:" is bold
  * For dischargers, changed "None" to "Not Specified"
* 470 - Added a link to the educators page
* 471 - Added a link on the waterbody report page

## Steps To Test:
1. Navigate to http://localhost:3000/community/auburn%20al/overview
2. Verify the waterbody conditions, in list view and map popups, are linked to the glossary
3. For each waterbody click on the "View Waterbody Report" link and verify the waterbody condition is linked to the glossary
4. Also verify there is a link at the top of the `State or Tribal Nation specific designated uses:` section on the waterbody report
5. Click on the Permitted Dischargers tab
6. Verify "None" is now "Not Specified"; check in filters list, list item, and map popup
7. Toggle off a couple of filters
8. Save the dischargers layer
9. In AGO verify the description has "Note:" bold
10. Navigate to http://localhost:3000/data
11. Verify the `How’s My Waterway January 2023 Watershed Academy Webcast` link is present and that it works
